prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -39,6 +42,39 @@ class FHIRRouter(
     queue: QueueAccess = QueueAccess,
 ) : FHIREngine(metadata, settings, db, blob, queue) {
 
+    /**
+     * Default Rules:
+     *   Must have message ID, patient last name, patient first name, DOB, specimen type
+     *   At least one of patient street, patient zip code, patient phone number, patient email
+     *   At least one of order test date, specimen collection date/time, test result date
+     */
+    val qualityFilterDefault: ReportStreamFilter = listOf(
+        "Bundle.entry.resource.ofType(MessageHeader).id.exists()",
+        "Bundle.entry.resource.ofType(Patient).name.family.exists()",
+        "Bundle.entry.resource.ofType(Patient).name.given.count() > 0",
+        "Bundle.entry.resource.ofType(Patient).birthDate.exists()",
+        "Bundle.entry.resource.ofType(Specimen).type.exists()",
+        "(Bundle.entry.resource.ofType(Patient).address.line.exists() or " +
+            "Bundle.entry.resource.ofType(Patient).address.postalCode.exists() or " +
+            "Bundle.entry.resource.ofType(Patient).telecom.exists())",
+        "(" +
+            "(Bundle.entry.resource.ofType(Specimen).collection.collectedPeriod.exists() or " +
+            "Bundle.entry.resource.ofType(Specimen).collection.collected.exists()" +
+            ") or " +
+            "Bundle.entry.resource.ofType(ServiceRequest).occurrence.exists() or " +
+            "Bundle.entry.resource.ofType(Observation).effective.exists())"
+    )
+
+    /**
+     * Default Rule:
+     *  Must have a processing mode id of 'P'
+     */
+    val processingModeFilterDefault: ReportStreamFilter = listOf(
+        "Bundle.entry.resource.ofType(MessageHeader).meta" +
+            ".extension('https://reportstream.cdc.gov/fhir/StructureDefinition/source-processing-id')" +
+            ".value.coding.code = 'P'"
```

Comment:
In this context, P is 'production'. The valid values per HL7 spec are 'D', 'T', and 'P' representing 'Debug', 'Test', and 'Production' respectively. The default filter for processing mode is to only allow 'production' data to pass through, but it can be overridden during onboarding to allow test data thought.